### PR TITLE
Hardware-1

### PR DESCRIPTION
**Pull request by YurkoWasHere**
Hardware purchase requisition

Pending

- [x] Agreement on layout of network ( [map](https://tomesh.net/map/?nodeURL=https://raw.githubusercontent.com/tomeshnet/node-list/cougar/tomeshnet-node-list.json&proposed=true&buildings=true) )
- [x] Confirmation of P2P antennas to use ( #98 ) 
- [x] Finalize estimated hardware requirements ([cloud]( https://cloud.tomesh.net/apps/files/?dir=/tomeshnet&fileid=480) )
- [x] Final quote from CDW

### CODE REVIEW

File: operations/hardware-budgets/hardware-1.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+| Duties    |           0.00 |
+| Taxes     |         615.16 |
+| **Total** |        5347.16 |
+
+## Sign-off
+
+_The sign-off should include at least one experienced member in the Network working group._
+
+Members of Network Planning, Design and Operations:
+- Yurko
```
Comment (YurkoWasHere) [inline operations/hardware-budgets/hardware-1.md:43]:
```suggestion
- @yurkowashere
```

File: operations/hardware-budgets/hardware-1.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+## Sign-off
+
+_The sign-off should include at least one experienced member in the Network working group._
+
+Members of Network Planning, Design and Operations:
+- @yurkowashere
+- Member 2
+
+Grant Administrators:
+- Yurko
```
Comment (YurkoWasHere) [inline operations/hardware-budgets/hardware-1.md:47]:
```suggestion
- @YurkoWasHere 
```

File: operations/hardware-budgets/hardware-1.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+| Taxes     |         615.16 |
+| **Total** |        5347.16 |
+
+## Sign-off
+
+_The sign-off should include at least one experienced member in the Network working group._
+
+Members of Network Planning, Design and Operations:
+- @yurkowashere
+- Member 2
```
Comment (Pedro-on-a-bike) [inline operations/hardware-budgets/hardware-1.md:44]: @Pedro-on-a-bike

Comment (Pedro-on-a-bike) [inline operations/hardware-budgets/hardware-1.md:44]: + -@Pedro-on-a-bike

**Review by Pedro-on-a-bike — APPROVED** [review]

File: operations/hardware-budgets/hardware-1.md (hunk 3 above)
Comment (darkdrgn2k) [reply operations/hardware-budgets/hardware-1.md:44]:
```suggestion
- @Pedro-on-a-bike
```

File: operations/hardware-budgets/hardware-1.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+_The sign-off should include at least one experienced member in the Network working group._
+
+Members of Network Planning, Design and Operations:
+- @yurkowashere
+- @Pedro-on-a-bike
+
+Grant Administrators:
+- @YurkoWasHere 
+- @Shrinks99
+- Administrator 3
```
Comment (benhylau) [inline operations/hardware-budgets/hardware-1.md:49]:
```suggestion
- @benhylau 
```

**Review by benhylau — APPROVED** [review]
To be clear, this means we are holding off on #100 bc we won't have enough to payout this and that.

File: operations/hardware-budgets/hardware-1.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
+# YYYY-MM-DD Hardware Budget for Toronto Community Network
```
Comment (benhylau) [inline operations/hardware-budgets/hardware-1.md:1]:
```suggestion
# 2020-10-01 Hardware Budget for Toronto Community Network
```

File: operations/hardware-budgets/hardware-1.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+Members of Network Planning, Design and Operations:
+- @yurkowashere
+- @Pedro-on-a-bike
+
+Grant Administrators:
+- @YurkoWasHere 
+- @Shrinks99
+- @benhylau 
+- Administrator 4
```
Comment (benhylau) [inline operations/hardware-budgets/hardware-1.md:50]:
```suggestion
- @TimTor
```

@TimTor has approved the PR. Adding him.

**Pull request merged**